doc/source/ray-air/examples/lightgbm_example.ipynb

```diff
@@ -71,13 +71,10 @@
     "from typing import Tuple\n",
     "\n",
     "import ray\n",
-    "from ray.train.lightgbm import LightGBMPredictor\n",
-    "from ray.data.preprocessors.chain import Chain\n",
-    "from ray.data.preprocessors.encoder import Categorizer\n",
+    "from ray.data import Dataset, Preprocessor\n",
+    "from ray.data.preprocessors import Categorizer, StandardScaler\n",
```

Comment:
Is there a particular reason to get rid of Chain? It's making the story for saving preprocessors state more complicated, and also calling fit_transform() multiple times is pretty inefficient.

Reply:
Yeah I was wondering the same thing actually. I think the original reason was because we wanted to just save the state of each individual preprocessor in the metadata, but since we're serializing the full preprocessor object it'll be preferred to keep `Chain`? cc @amogkam

Reply:
Going to un-deprecate `Chain`, but leave it undocumented. It doesn't have any performance benefits (since under the hood it just calls `fit_transform` for each preprocessor) but it might have some usability benefits of not having to call `fit_transform`/`serialize`/`deserialize` for each preprocessor.